### DISCUSSION TITLE
对于长训轮之类请求的错误疑问

### DISCUSSION THREAD

**Discussion by zcl1115** [harmonycloud/kindling#616] [agent collector] [answered]
采集发现对于apollo的请求，类似/notifications/v2的longpoll请求，没有办法第一时间返回响应，而是进行了挂起，我发现network_analyzer里面对于没有响应体的请求，会直接设置为错误请求，能不能优雅的过滤掉这类请求？还是说我只能在代码里面暴力的过滤apollo的请求端口为错误

**Comment by dxsup** [comment, marked as answer]
目前确实不能优雅地识别这类请求。

要解决这类问题，一种做法是新增一个配置项endpoints列表，标识这个列表中的endpoints是long poll请求，然后在分析请求时不要把他们设置为错误请求。
